package.json

```diff
@@ -66,7 +67,8 @@
     "prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme",
     "test": "jest --testPathPattern=tests",
     "version": "oclif readme && git add README.md",
-    "test:unit": "mocha --forbid-only -r ts-node/register \"test/**/*.test.ts\""
+    "test:unit": "mocha --forbid-only -r ts-node/register \"test/**/*.test.ts\"",
```

Comment:
We Can Remove the Unit Test commands, because it has the graph QL tests which are already present in the Integration tests

Reply:
Removed the unit test script and files. Thanks Zuhair